src/main/resources/db/scripts/V7_CREATE_ORDER_TABLE.SQL

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE order (
+    orderId INT PRIMARY KEY AUTO_INCREMENT,
```

Comment:
Here for database columns: 
  instead of orderId use order_id.
But in java code we can use orderId.
It will work seemless.